### PR TITLE
all the llamas in canary

### PR DESCRIPTION
**Pull request by msaroufim**
All of these OOM on a single device but want to make them all available for the distributed tests @H-Huang is working on

### CODE REVIEW

**Review by xuzhao9 — APPROVED** [review]
More llamas!

File: torchbenchmark/canary_models/llama_v2_13b/__init__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from torchbenchmark.tasks import NLP
```
Comment (H-Huang) [inline torchbenchmark/canary_models/llama_v2_13b/__init__.py:1]: just curious, these models are in `canary_models` as opposed to `models` dir. Does this effect how these models are pulled in either from torchbench or dyanmorunner and if so in what way?

**Review by H-Huang — APPROVED** [review]
Awesome! Thanks for adding :)

File: torchbenchmark/canary_models/llama_v2_13b/__init__.py (hunk 1 above)
Comment (msaroufim) [reply torchbenchmark/canary_models/llama_v2_13b/__init__.py:1]: This error message explains it super well, should be no difference

```
  File "/var/lib/jenkins/workspace/benchmarks/dynamo/torchbench.py", line 302, in load_model
    raise ImportError(f"could not import any of {candidates}")
ImportError: could not import any of ['torchbenchmark.models.stable_diffusion', 'torchbenchmark.canary_models.stable_diffusion', 'torchbenchmark.models.fb.stable_diffusion']
ERROR
```

**Pull request closed without merging**